arbigent-ai-anthropic/src/main/java/io/github/takahirom/arbigent/AnthropicAi.kt

```
@@ -0,0 +1,947 @@
+      system = systemContents,
+      messages = messages,
+      tools = toolDefinitions,
+      toolChoice = AnthropicToolChoice.Any,
+    )
+    val responseText = try {
+      createMessage(
+        requestUuid = requestUuid,
+        request = messagesRequest,
+        aiOptions = decisionInput.aiOptions
+      )
+    } catch (e: AnthropicAiRateLimitExceededException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)